pkg/controller/vsphere/reconciler.go

```diff
@@ -560,6 +576,18 @@ func clone(s *machineScope) (string, error) {
 		return "", handleVSphereError(multipleFoundMsg, notFoundMsg, defaultError, err)
 	}
 
+	hwVersion, err := getHwVersion(s.Context, vmTemplate)
+	if err != nil {
+		// TODO throw an event also?
+		klog.Warningf("Unable to detect machine template HW version for machine %s: %v", s.machine.GetName(), err)
+	}
+	if hwVersion == 13 { // TODO figure out this condition more precisely
+		return "", machineapierros.InvalidMachineConfiguration(
+			"Hardware version 13 is not supported, clone stopped. " +
+				"Please update machine template: https://access.redhat.com/articles/6090681",
```

Comment:
I'll work with docs to get an update for this knowledge article.  Thinking it should live here: https://docs.openshift.com/container-platform/4.10/updating/updating-hardware-on-nodes-running-on-vsphere.html .  For now, knowledge article is fine.